### PR TITLE
xclbinutil - development add fingerprint support in aie_partition

### PR DESCRIPTION
**Pull request by xfreid**
#### Problem solved by the commit
Add fingerprint support in AIE_PARTITION

#### Bug / issue (if any) fixed, which PR introduced the bug, how it was discovered
N/A

#### How problem was solved, alternative solutions (if any) and why they were rejected
N/A

#### Risks (if any) associated the changes in the commit
None

#### What has been tested and how, request additional testing if necessary
Manual and unit test

#### Documentation impact (if any)
None

### CODE REVIEW

**Comment by larry9523** [comment]
ok to test

File: src/runtime_src/core/include/xclbin.h [hunk 1]
```diff
@@ -570,11 +570,13 @@ extern "C" {
...
         uint32_t operations_per_cycle;      // Operations per cycle. Used later to create TOPS (operations_per_cycle * <AIE Clock Frequency>)
         uint8_t padding[4];
+        uint64_t inference_fingerprint;
+        uint64_t pre_post_fingerprint;
         struct aie_partition_info info;     // Partition information
         struct array_offset aie_pdi;        // PDI Array (aie_partition_info)
         uint8_t reserved[54];               // Reserved
     };
-    XCLBIN_STATIC_ASSERT(sizeof(struct aie_partition) == 168, "aie_partition structure no longer is 168 bytes in size");
+    XCLBIN_STATIC_ASSERT(sizeof(struct aie_partition) == 184, "aie_partition structure no longer is 168 bytes in size");
```
Comment (rozumx) [inline src/runtime_src/core/include/xclbin.h:579]: The error string doesn't represent the number of bytes being checked.
```suggestion
    XCLBIN_STATIC_ASSERT(sizeof(struct aie_partition) == 184, "aie_partition structure no longer is 184 bytes in size");
```

File: src/runtime_src/core/include/xclbin.h [hunk 2]
```diff
@@ -570,11 +570,13 @@ extern "C" {
         uint32_t mpo_name;                  // Name of the aie_partition 
         uint32_t operations_per_cycle;      // Operations per cycle. Used later to create TOPS (operations_per_cycle * <AIE Clock Frequency>)
         uint8_t padding[4];
+        uint64_t inference_fingerprint;
```
Comment (rozumx) [inline src/runtime_src/core/include/xclbin.h:573]: Could you add a one line comment description describing what this variable represents.
```suggestion
        uint64_t inference_fingerprint;  // The unique hash value of the inference function
```

File: src/runtime_src/core/include/xclbin.h [hunk 3]
```diff
@@ -570,11 +570,13 @@ extern "C" {
...
+        uint64_t pre_post_fingerprint;
```
Comment (rozumx) [inline src/runtime_src/core/include/xclbin.h:574]: Could you add a comment here as well.

**Review by rozumx — CHANGES_REQUESTED** [review]

File: src/runtime_src/core/include/xclbin.h (hunk 3 above)
Comment (xfreid) [reply src/runtime_src/core/include/xclbin.h:574]: add comment for pre_post_fingerprint

**Review by rozumx — APPROVED** [review]

**Pull request merged**